languages/go/exercises/concept/conditionals/.meta/design.md

```diff
@@ -0,0 +1,34 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student the basics of the Concept of Conditionals in Go.
+
+## Learning objectives
+
+- Know of the existence of the `bool` type.
+- Know how to define an if statement.
+- Know to avoid else after return.
+- Know how to define a switch statement.
+
+## Out of scope
+
+- type-switch
+- iterations
+- continue (as there are no iterations)
+
+## Concepts
```

Comment:
I'm suggesting a slightly normalized version of the concepts used below, to make things more consistent cross-tracks.

Reply:
corrected